src/app/static/src/tests/components/accessibility.test.ts

```diff
@@ -0,0 +1,118 @@
+import {emptyState} from "../../app/root";
+import Vuex from "vuex";
+import Accessibility from "../../app/components/Accessibility.vue"
+
+import registerTranslations from "../../app/store/translations/registerTranslations";
+import {shallowMount} from "@vue/test-utils";
+import {expectTranslated} from "../testHelpers";
+import {actions} from "../../app/store/root/actions";
+import {mutations} from "../../app/store/root/mutations";
+import {Language} from "../../app/store/translations/locales";
+
+describe(`accessibility`, () =>{
+    const createStore = () => {
+        const store = new Vuex.Store({
+            state: emptyState(),
+            actions: actions,
+            mutations: mutations
+        });
+        registerTranslations(store);
+        return store;
+    };
+
+    const getWrapper = () => {
+        return shallowMount(Accessibility,
+            {
+                store: createStore()
+            })
+    }
+
+    it(`renders accessibility tags as expected`, () => {
+        const rendered = getWrapper()
+        expect(rendered.find("#accessibility-content")
+            .find("h1").text()).toBe("Accessibility on Naomi")
+
+        const h2 = rendered.find("#accessibility-content").findAll("h2")
+
+        expect(h2.at(0).text()).toBe("How accessible the website is")
```

Comment:
Was it agreed that only selected parts of the content needed to be tested as I think some is missing?

Reply:
Probably don't need to test every word but I suppose it might be include to include some 'startsWith' tests for the paragraphs.

Reply:
This page is static, I have tested main tags.